provisioning/base/resources.tf

```diff
@@ -0,0 +1,50 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+variable "environment" {
```

Comment:
How is this app handling environment variables? Other lambdas that use terraform use something like this: 

`# Load ENV vars from ./config/{environment}.env
  environment {
    variables = { for tuple in regexall("(.*?)=(.*)", file("../../config/${var.environment}.env")) : tuple[0] => tuple[1] }
  }`

Reply:
Looks like previously app config was only stored in `.travis.yml`, which is removed in this PR. The env vars should prob be moved to `./config/*.env` files and loaded in tf as @danamansana suggests above.